src/actions/appointments.ts

```diff
@@ -0,0 +1,23 @@
+export function getAppointments() {
```

Comment:
Please rename this folder from 'action' to smth else (like mock-data, or mock-helper)